test/auto/injectorSpec.js

```diff
@@ -237,6 +237,15 @@ describe('injector', function() {
     });
 
 
+    // Only Chrome and Firefox support this syntax.
+    if (/chrome|firefox/i.test(navigator.userAgent)) {
+      it('should be possible to annotate functions that are declared using ES6 syntax', function() {
+        // The function is generated using `eval` as just having the ES6 syntax can break some browsers.  
+        expect(annotate(eval('({ fn(x) { return; } })').fn)).toEqual(['x']);
```

Comment:
Which JS parser? What browser versions are you talking about? Are you talking about the test file failing to parse in older browsers? Or what?